PacketDotNet/Interfaces/ISourceDestinationPort.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace PacketDotNet.Interfaces
+{
+    public interface ISourceDestinationPort
```

Comment:
This one feels weird. Typically one would add an interface so they could treat packets in a similar manner to another so store an array of Tcp and Udp packets and then look at their source and destination, or provide a function or class that could handle an ISourceDestinationPort and users could use it for both Tcp and Udp packets. What is the end goal here?